### ISSUE TITLE
test failures in -4.1.2 under py3.4; test_rdfpipe_bytes_vs_str, test_rdfpipe_mdata_open, testHTML

### ISSUE THREAD

**Issue by idella** [RDFLib/rdflib#396]
with # python -V 
Python 3.4.0

```python
======================================================================
FAIL: Issue 375: rdfpipe command generates bytes vs. str TypeError
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python3.4/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/mnt/gen2/TmpDir/portage/dev-python/rdflib-4.1.2/work/rdflib-4.1.2-python3_4/build/src/test/test_issue375.py", line 163, in test_rdfpipe_bytes_vs_str
    assert res.strip() == rdfa_expected
AssertionError

======================================================================
FAIL: Issue 375: rdfa1.1 and mdata processors used file() builtin
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python3.4/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/mnt/gen2/TmpDir/portage/dev-python/rdflib-4.1.2/work/rdflib-4.1.2-python3_4/build/src/test/test_issue375.py", line 183, in test_rdfpipe_mdata_open
    assert res == mdata_expected
AssertionError

======================================================================
FAIL: test.test_xmlliterals.testHTML
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python3.4/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/mnt/gen2/TmpDir/portage/dev-python/rdflib-4.1.2/work/rdflib-4.1.2-python3_4/build/src/test/test_xmlliterals.py", line 77, in testHTML
    assert l2.value is not None, 'xml must have been parsed'
AssertionError: xml must have been parsed

----------------------------------------------------------------------
Ran 2314 tests in 45.413s

FAILED (SKIP=99, failures=3)
Running nose with: --attr= test rdflib --where=./ --with-doctest --doctest-extension=.doctest --doctest-tests

```

Do you get these? Do you require anything else?

**Comment by joernhees** [comment]
can you try again with the latest dev version? i think part of the issue might have been fixed in #477

**Comment by joernhees** [comment]
reopen if still existing
